### DISCUSSION TITLE
How to make the alarm card to do not ask for code

### DISCUSSION THREAD

**Discussion by lbreggi** [matt8707/ha-fusion#289] [Q&A]
Hi! this is by far the best pre designed dashboard for HA, congrats all the people involved in the project!
Just a question: Is there a way I can make the alarm card to do not ask for a code (similar to the one in mushroom)?
many thanks!

**Comment by abeksis** [comment]
Hi

The option is currently unavailable. I'm sure such an option will be added later.

**Reply by lbreggi** [reply]
Thanks a lot! I will be looking forward for it!
